cifar10-multiannotator/multiannotator_labels.ipynb

```diff
@@ -0,0 +1,4840 @@
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "Bringing Your Own Data (BYOD)?\n",
+    "\n",
+    "You can easily replace the above with your own multiannotator dataset, and continue with the rest of the example.\n",
+    "\n",
+    "`multiannotator_labels` should be a numpy array or pandas DataFrame with each column representing an annotator and each row representing an example. Your classes (and entries of `multiannotator_labels`) should be represented as integer indices 0, 1, ..., num_classes - 1, where examples that are not annotated by a particular annotator are represented using `np.nan`.\n",
+    "    \n",
+    "If working with images, `image_paths` should be a string of absolute or relative paths to the where each index corresponds the example for that row of `multiannotator_labels`.\n",
+    "\n",
+    "If working with other data, `image_paths` should be a string of examples corresponding to the row of `multiannotator_labels`.\n",
+    "\n",
+    "</div>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "66bafbf8",
+   "metadata": {},
+   "source": [
+    "Before training our machine learning model, we must first obtain the consensus labels from the annotators that labeled the data. The simplest way to obtain an initial set of consensus labels is to select it using majority vote."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "5a953e3a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "consensus_labels = get_majority_vote_label(multiannotator_labels)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3c114734",
+   "metadata": {},
+   "source": [
+    "## 3. Train model and use cleanlab to get better consensus labels"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e31453ef",
+   "metadata": {},
+   "source": [
+    "Next, we will train our model on the consensus labels obtained using majority vote to compute out-of-sample predicted probabilities. We will then use these `pred_probs` to generate more informed `consensus_labels` using Cleanlab's [CROWDLAB](link to something) algorithm. We them use these `consensus_labels` to train a better model that generates more accurate `pred_probs`. This process itterates until the `consensus_labels` have no more improvement.\n",
+    "\n",
+    "`train_model()` trains a `resnet18` image model using cross validation to get out-of-sample predicted probabilities on the whole dataset. The function can be replaced with a custom training algorithm.\n",
+    "\n",
+    "We use the [AutoGluon](https://auto.gluon.ai/) library to train powerful image classifiers (Residual Network or Swin Transformer), but this requires a GPU to run in a reasonable time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "370c66ca",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_type = \"swin_base_patch4_window7_224\" # You can also try with \"swin_base_patch4_window7_224\"\n",
```

Comment:
I believe this is inconsistent with the text above that says "`train_model()` trains a `resnet18` image model"